### ISSUE TITLE
`logit_scale` seems to stay at its initial value in released DinoTxt weights

### ISSUE THREAD

**Issue by KuangHaofei** [facebookresearch/dinov2#548]
Dear authors,

Thanks for your nice work and for releasing the code and weights of the dino.txt model!

I noticed that recent provided pre-trained weights do not seem to contain the `logic_scale` parameters. 

When I load the pretrained DinoTxt model via [dinov2.hub.dinotxt.dinov2_vitl14_reg4_dinotxt_tet1280d20h24l()](https://github.com/facebookresearch/dinov2/blob/592541c8d842042bb5ab29a49433f73b544522d5/dinov2/hub/dinotxt.py#L8), the parameter [model.logit_scale](https://github.com/facebookresearch/dinov2/blob/592541c8d842042bb5ab29a49433f73b544522d5/dinov2/hub/text/dinotxt_model.py#L68) is always equal to its initial value `math.log(1/0.07)` and does not seem to have been updated during training.

Meanwhile, I noticed that the `DinoTxt` already provides a convenience method:
```python
def get_logits(self, image, text):
    text_features  = self.encode_text(text,   normalize=True)
    image_features = self.encode_image(image, normalize=True)
    image_logits = self.logit_scale.exp() * image_features @ text_features.T
    text_logits  = image_logits.T
    return image_logits, text_logits
```
So if a trained logit_scale were stored in the checkpoint, we could simply call `model.get_logits(...)` and obtain the correct temperature automatically. But in [notebooks/dinotxt.ipynb](https://github.com/facebookresearch/dinov2/blob/main/notebooks/dinotxt.ipynb) the logits are computed manually with the hard-coded `logit_scale = 100` as:
```python
logits = 100. * image_features @ zeroshot_weights
```
instead of using `model.get_logits()` or `model.logit_scale.exp()`. 

So I would like to ask: 
1. Was logit_scale frozen during DinoTxt training, or filtered out by the optimizer?
2. If it was actually trained, could future checkpoints include the learned value?
3. Is the hard-code `logit_scale = 100` sufficient to handle most scenarios?

Again, thank you for the excellent work and for considering these questions!

**Comment by cijose** [comment]
Thank you and sorry for the omission of logit_scale from  the checkpoints.  Please find answers to your questions as follows:  

> 1. Was logit_scale frozen during DinoTxt training, or filtered out by the optimizer?
The logit_scale is not frozen during training, it is updated and clipped at  a value of 100.
> 3. If it was actually trained, could future checkpoints include the learned value?
Yes we will include it in the future checkpoints, thanks for the feedback
> 5. Is the hard-code `logit_scale = 100` sufficient to handle most scenarios?
Yes because logit scale is 100 for the trained checkpoint

Kind regards
